### DISCUSSION TITLE
Storing Typescript Interface Object

### DISCUSSION THREAD

**Discussion by saikarthikp9** [prisma/prisma#9813] [Q&A] [answered]
I have an interface as below:
```ts
export interface Order {
    subject: string;
    data: {
        order_id: string;
        user_id: string;
    };
}
```
I want to save the data property as a column. Should this column be Json or String type?
I want to be able to retrieve it and access it as: data.order_id

Currently I am not able to do this:
```ts
db.order.create({
			data: {
				subject: "asdf",
				data: {
                                      order_id: "asdf";
                                      user_id: "asdf";
                                  }
			}
		})
```
Please advice

**Comment by ryands17** [comment, marked as answer]
@saikarthikp9 👋 

Best to store this in a Json field and typecast it to any when storing it so that TypeScript doesn't complain. Currently we are working on directly specifying Json types in the schema so it would be great if you could add a 👍 to this request: https://github.com/prisma/prisma/issues/3219

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
